### PR TITLE
Update agent markdown

### PR DESCRIPTION
**Pull request by RabsonJ**

### CODE REVIEW

**Review by mohsinenar — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,25 +1,51 @@
...
+
+_Tsunami is a general purpose network security scanner with an extensible plugin system for detecting high severity vulnerabilities with high confidence._
+
+---
+
+<p align="center">
+<img src="./agent_tsunami.png" alt="agent-tsunami" />
+</p>
+
+An implementation of [Ostorlab Agent](https://pypi.org/project/ostorlab/) for the [Tsunami Scanner](https://github.com/google/tsunami-security-scanner) by Google.
```
Comment (3asm) [inline README.md:19]: What is an implementation of what is described?

I suggest adding: This repo is xxx.

File: README.md [hunk 2]
```diff
@@ -1,25 +1,51 @@
...
 
 ## Usage
+Refer to the [Ostorlab Documentation](https://github.com/Ostorlab/ostorlab/blob/main/README.md)
```
Comment (3asm) [inline README.md:22]: I think a basic command to perform scan here won't hurt, and refer to the documentation.

File: README.md [hunk 3]
```diff
@@ -1,25 +1,51 @@
...
+
+ ### Install directly from ostorlab agent store
```
Comment (3asm) [inline README.md:24]: We need an introductory sentence to mention that agent can either be directly download from the store or built.

File: README.md [hunk 4]
```diff
@@ -1,25 +1,51 @@
...
+
+`pip3 install ostorlab`
+
+ 2. Clone this repository.
+
 `git clone https://github.com/Ostorlab/agent_tsunami.git && cd agent_tsunami `
-   
- 4. build the agent image using ostorlab cli.
+
+ 3. Build the agent image using ostorlab cli.
```
Comment (3asm) [inline README.md:39]: If the agent is built, we need to explain how to run it.

File: README.md [hunk 5]
```diff
@@ -1,25 +1,51 @@
...
 
- ### Install directly from ostorlab agent store.
- 
- **soon.**
+
+## Contributing
+
+### Contributing Guide
+
+Before making any pull request, please make sure to read the [Contributing Guide](#) to learn about our development process and how to propose bugfixes and improvements.
```
Comment (3asm) [inline README.md:48]: We have one?

**Review by 3asm — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -1,25 +1,70 @@
...
+`ostorlab scan run --install --agents agent/ostorlab/tsunami ip 8.8.8.8`
+
+This command will download and install `agent/ostorlab/tsunami` and target the ip `8.8.8.8`.
+For more information, please refer to the [Ostorlab Documentation](https://github.com/Ostorlab/ostorlab/blob/main/README.md)
+
 
 ## Usage
 
-Refer to Ostorlab documentation.
+Agent Tsunami can be installed directly from the ostorlab agent store or built from the repository.
```
Comment (3asm) [inline README.md:31]:
```suggestion
Agent Tsunami can be installed directly from the ostorlab agent store or built from this repository.
```

**Review by 3asm — APPROVED** [review]

**Pull request merged**